src/db/special_ability_phase_stat_effects_tables_vco.tsv

```diff
@@ -0,0 +1,7 @@
+phase	value	stat	how
+#special_ability_phase_stat_effects_tables;3;db/special_ability_phase_stat_effects_tables/vco			
+vco_lord_abilities_youre_next	0.5500	scalar_speed	mult
```

Comment:
alphabetical order

Reply:
I am not sure why are we keeping the `vco_lord_abilities_youre_next` ability. Are we using it? I think we are just using `vco_lord_abilities_hunter_of_champions`.

Reply:
Correct, removed.